test/e2e/tools.mjs

```diff
@@ -39,6 +42,95 @@ export async function getStats(url) {
     return fs.readJSON(path);
 }
 
+/**
+ * @param {string | URL} url
+ */
+export function getActorTestDir(url) {
+    const __filename = fileURLToPath(url);
+    const __dirname = dirname(__filename);
+    return join(__dirname, 'actor');
+}
+
+/**
+ * @param {string} dirName
+ */
+export async function runActor(dirName) {
+    let stats;
+    let datasetItems;
+
+    if (process.env.npm_config_platform) {
+        await copyPackages(dirName);
+        // TODO: add some check for token (or check that we are logged in)
+        await exec('npx -y apify-cli push', { cwd: dirName });
+
+        const actorName = await getActorName(dirName);
+        const client = new ApifyClient({ token: process.env.APIFY_TOKEN });
+        const { items: actors } = await client.actors().list();
+        const { id } = actors.find((actor) => actor.name === actorName);
+
+        const { defaultKeyValueStoreId, defaultDatasetId } = await client.actor(id).call();
+        const { value } = await client.keyValueStore(defaultKeyValueStoreId).getRecord('SDK_CRAWLER_STATISTICS_0');
+        stats = value;
+        const { items } = await client.dataset(defaultDatasetId).listItems();
+        datasetItems = items;
+    } else {
+        await exec('npx -y apify-cli run -p', { cwd: dirName });
+        stats = await getStats(dirName);
+        datasetItems = await getDatasetItems(dirName);
+    }
+
+    return { stats, datasetItems };
+}
+
+/**
+ * @param {string} dirName
+ */
+async function getActorName(dirName) {
+    const actorPackageFile = await fs.readJSON(join(dirName, 'package.json'));
+    return actorPackageFile.name;
+}
+
+/**
+ * @param {string} dirName
+ */
+async function copyPackages(dirName) {
```

Comment:
Added annotation there + updated browser-based actors Dockerfiles to use multistage build. We could update all Dockerfiles to use multi-stage, but I don't see any reason to do it now (since other actors use `actor-node` image, which is using root and thus doesn't have any issues).